src/job/jobHandler.ts

```diff
@@ -303,6 +317,12 @@ export abstract class JobHandler {
         const makeCommandsResp = await this._commandExecutor.execute([command]);
         await this.logBuildDetails(makeCommandsResp);
       }
+
+      // Call Gatsby Cloud preview webhook after persistence module finishes for staging builds
+      const isFeaturePreviewWebhookEnabled = process.env.GATSBY_CLOUD_PREVIEW_WEBHOOK_ENABLED?.toLowerCase() === 'true';
+      if (key === 'persistence-module' && this.name === 'Staging' && isFeaturePreviewWebhookEnabled) {
+        await this.callGatsbyCloudWebhook();
```

Comment:
I'd rather not expose this call as another Makefile rule (plus we may not want to expose the user's site ID). I thought about potentially allowing the executor be compatible with both `make` commands and function calls, but it seemed like a bigger change than necessary for the scope of this PR